image_width: 640
image_height: 480
camera_name: left_camera
camera_matrix:
  rows: 3
  cols: 3
  data: [688.2349727874704, 0.0, 357.8519494049359, 0.0, 692.2819006046751, 233.01648485365695, 0.0, 0.0, 1.0]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [-0.15340634859674976, -0.10730062749709922, -0.0025286730548697506, -0.0010114921129803943, 0.0]
rectification_matrix:
  rows: 3
  cols: 3
  data: [0.9993048009296369, -0.0009609375907641417, 0.037269175439297406, 0.0008373971140205948, 0.9999941039672087, 0.003330284806577928, -0.0372721558948755, -0.00329676049572156, 0.9992997136821274]
projection_matrix:
  rows: 3
  cols: 4
  data: [683.842068928247, 0.0, 343.5896682739258, 0.0, 0.0, 683.842068928247, 228.71416473388672, 0.0, 0.0, 0.0, 1.0, 0.0]
